### DISCUSSION TITLE
Show more info on machine when installing new game server.

### DISCUSSION THREAD

**Discussion by birdman3131** [gameserverapp/Platform#501] [Suggestions]
When you go to install a new game server it would be great if you had some more stats on each dediconnect.

Strongly useful:
Storage space left.
Current memory in use/total memory.

Potential others to include.
Number of running game servers on each machine. (Helpful when combined with the memory in use.)
Estimated initial install size of server. (As seen by GSA in the last 50 installs.)

**Comment by mvaessen** [comment]
Thanks for your suggestion, this is now available on the dashboard!
